examples/openshift/clamav/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM openshift/origin:latest
```

Comment:
Discussion: Should this clam deamon be part of image-inspector? If it is then we need more documentation on it either in the main README or in a local one.